apps/expo/src/components/Heading.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Text, TouchableOpacity, View } from "react-native";
+
+import Bookmark from "../../assets/bookmark.svg";
+import LeftArrow from "../../assets/left-arrow.svg";
+
+interface HeadingProps {
+  leftArrow?: {
+    onPress: () => void;
+  };
+  bookmark?: {
+    onPress: () => void;
+  };
```

Comment:
can you make it so that you can insert your own svg for the icons -- there are some headings that don't use bookmark and use something else